CHANGELOG.md

```diff
@@ -2,6 +2,45 @@
 
 ## unreleased
 
```

Comment:
We need to add a new section:
```
### Breaking Changes
```

Reply:
you can use pivotal to search for tasks with a `breaking` tag. We should also use that tag in PRs.
cc: @amityadav0 , @man-of-code

Reply:
ok